### ISSUE TITLE
basic-tutorial.md -- name should not equal server_name

### ISSUE THREAD

**Issue by rebeccc** [pythongssapi/python-gssapi#208]
### What went wrong?
On line 85 of basic-tutorial.md, the call to Credentials fails because it is server_name and not server_hostbased_name

### How do we reproduce?
Based on the tutorial, you set server_name and server_hostbased_name as follows:
```
server_hostbased_name = gssapi.Name('HTTP@' + FQDN, name_type=gssapi.NameType.hostbased_service)
server_name = gssapi.Name('HTTP/sross@')
```
and then make a call to Credentials using server_name rather than server_hostbased_name, which causes it to fail.
```
server_creds = gssapi.Credentials(usage='accept', name=server_name)
```

**Comment by rebeccc** [comment]
There are a couple of other places in the tutorial that uses server_name instead of server_hostbased_name. I've made a list of the ones I realized wouldn't work. 

this one is just in a sequence of checks and doesn't fail, but will show up false.
line 103:
```
server_creds.name == server_name
```
line 128:
```
client_ctx = gssapi.SecurityContext(name=server_name, usage='initiate')
```

**Comment by frozencemetery** [comment]
Fixed by #209
